include/button_state.h

```diff
@@ -4,6 +4,8 @@
 enum class ButtonState
 {
   NEUTRAL,
+  // It is advised to treat HOLDING the same as NEUTRAL
+  // as it is only used internally.
```

Comment:
This should also be done for `include/joystick_position.h`. While you're at it, you can also rename it to `joystick_state.h` for consistency.